x/ecocredit/params.go

```diff
@@ -149,7 +149,7 @@ func DefaultParams() Params {
 	return NewParams(
 		sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, DefaultCreditClassFeeTokens)),
 		[]string{},
-		false,
+		true,
```

Comment:
@clevinson Do you think  this is a good way to go?

Reply:
I believe we want to keep in `false` by default (if anyone out of regen wants to use the ecocredit module), but in our regen-ledger, we set it to `true` (see above comment). Is that right @clevinson ?

Reply:
yes, i believe this also makes it easier for testing internally